gateway/1.4/modules/ROOT/nav.adoc

```diff
@@ -13,7 +13,7 @@
 * xref:flex-gateway-k8-getting-started.adoc[]
 * xref:flex-review-prerequisites.adoc[Reviewing Prerequisites]
 * xref:flex-gateway-set-up.adoc[]
-** xref:flex-install.adoc[Install Flex Gateway]
+** xref:flex-install.adoc[]
```

Comment:
We don't need to add the title of the page unless it differs from the actual title on the page, which it doesn't/won't.

Reply:
Hi @dukesphere, I included the title here in the left nav because I wanted it to be imperative (instead of a gerund), to conform with the other titles in the left nav section. It will be strange if one is a gerund and the others are imperative. 

Maybe the title of the install page it links to should be imperative?

It's a different story altogether if they should all be gerunds. That's a can of worms. :)

Reply:
Oh, I had changed the title to be imperative. Then we want page titles to be gerunds and the nav to be imperative? @chris-campbell9

Reply:
@dukesphere Currently all the topic titles within buckets are imperative. The buckets themselves are gerunds. And page titles are gerunds.

This was the guidance when we started Flex. I've recently come to understand that this has changed, and should probably be addressed some time in the future.

But IMO in the short-term we should be consistent.

Reply:
Ok, I'll go back and fix this. Thx for noticing. I should have noticed but obviously didn't. 😆

Reply:
done